### DISCUSSION TITLE
Can't get contabo S3 object storage to work

### DISCUSSION THREAD

**Discussion by SeleDreams** [misskey-dev/misskey#11403] [Q&A]
Hi, 
I've been managing my own misskey instance on my server and it's working well, but because my server has only 100gb of storage, I got a 1tb object storage from contabo.
however I can't seem to get it working properly, I've properly configured my access key and my private key, I have set the bucket as public, and I've used the public link of my bucket, but nothing seems to work, when I upload a file to my drive, the file does *not* appear in the bucket and downloading it leads to an url leading to the bucket, but it returns an access denied since the file never got uploaded at the url.

I have created my instance using docker, I am not really sure if there's some kind of log of th object storage that could help me find the reason of the failure.

**Comment by xxdd101** [comment]
好问题
